source/solvers/output_struct.cc

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#include <solvers/output_struct.h>
+
+#include <deal.II/base/vector_tools>
```

Comment:
Do you need this include? I would presume what you need is the global vector type include and the dealii parallel vector one